package dos;
import "dos.proto";


option cc_generic_services = true;
option py_generic_services = true;

message RunContainerRequest {
  optional string name = 1;
  optional Container container = 2;
}

message RunContainerResponse {
  optional RpcStatus status = 1;
}

message ShowContainerRequest {}
message ShowContainerResponse {}

service Engine {
  rpc RunContainer(RunContainerRequest) returns(RunContainerResponse);
}
